Code/Tools/Standalone/Source/LUA/LUAEditorMainWindow.cpp

```diff
@@ -346,7 +346,11 @@ namespace LUAEditor
             {
                 auto selectedAsset = selectedAssets.front();
                 const AZStd::string filePath = selectedAsset->GetFullPath();
-                EBUS_EVENT(Context_DocumentManagement::Bus, OnLoadDocument, filePath, true);
+                auto entry_type = selectedAsset->GetEntryType();
+                if (entry_type == AzToolsFramework::AssetBrowser::AssetBrowserEntry::AssetEntryType::Source)
```

Comment:
standards: The variable naming scheme should be [lowerCamelCase](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#variable-names) according to the Coding Standards

Reply:
Apologies. This has now been fixed.